cmd/vsphere-cloud-controller-manager/main.go

```diff
@@ -210,148 +169,24 @@ func main() {
 	}
 }
 
-// initFunc is used to launch a particular controller.  It may run additional "should I activate checks".
```

Comment:
this deleted code was originally copied from the cloud provider repo to bypass upstream bug, as the upstream fix merged, we could directly using it from the cloud provider repo

Reply:
sounds good, thanks for the context!